go-controller/pkg/ovn/management-port.go

```diff
@@ -38,14 +39,35 @@ func configureManagementPort(nodeName, clusterSubnet, routerIP, interfaceName, i
 
 	// Create a route for the entire subnet.
 	_, err = exec.Command("ip", "route", "add", clusterSubnet, "via", routerIP).CombinedOutput()
-	return err
+	if err != nil {
+		return err
+	}
+
+	if clusterServicesSubnet != "" {
+		// Flush the route for the services subnet (in case it was added before).
+		_, err = exec.Command("ip", "route", "flush",
+			clusterServicesSubnet).CombinedOutput()
+		if err != nil {
+			return err
+		}
+
+		// Create a route for the services subnet.
+		_, err = exec.Command("ip", "route", "add", clusterServicesSubnet,
```

Comment:
Should it be a via rule? I end up using the 'dev' rule directly. Let me test.
And perhaps we could use the netlink library (subsequent PR).

Reply:
Just a few lines above this change, we use the same command for clusterSubnet to be reachable via node (for kubelet health-check). This uses the same method.